### ISSUE TITLE
Content shown on hover should be dismissible

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7253]
E.g. by pressing escape.

We have menus on at the top on the page and the `Sort by` drop down menu.

https://www.w3.org/WAI/WCAG22/Techniques/client-side-script/SCR39.html

**Comment by szakarias** [comment]
![image](https://github.com/dart-lang/pub-dev/assets/3865266/c5819361-5aa9-4796-87b2-a453973658fe)

**Comment by sigurdm** [comment]
![image](https://github.com/dart-lang/pub-dev/assets/8613953/78c53947-f431-49bd-a810-8f6c405a5e81)
